src/content/ja/2019/cdn.md

```diff
@@ -0,0 +1,1565 @@
+---
+part_number: IV
+chapter_number: 17
+title: CDN
+description: CDNの採用と使用法、RTTとTLSの管理、HTTP/2の採用、キャッシング、および共通ライブラリとコンテンツCDNをカバーする2019 Web年鑑のCDNの章。
+authors: [andydavies, colinbendell]
+reviewers: [yoavweiss, paulcalvano, pmeenan, enygren]
+translators: [ksakae]
+discuss: 1772
+published: 2019-11-11T00:00:00.000Z
+last_updated: 2019-12-04T00:00:00.000Z
+---
+
+## 導入
+
+「CDNの使用」は、Webサイトの読み込みを高速化するための[Steve Soudersの最初の推奨事項](http://stevesouders.com/hpws/rules.php)の1つでした。今日でも有効なアドバイスです。Web年鑑のこの章ではSteveの推奨事項がどの程度広く採用されているか、サイトがコンテンツ配信ネットワーク（CDN）をどのように使用しているか、およびそれらが使用している機能のいくつかを検討します。
+
+基本的にCDNは待ち時間（パケットがネットワーク上の2つのポイント間、たとえば訪問者のデバイスからサーバーに移動する時間）を短縮します、待ち時間はページの読み込み速度の重要な要素です。
+
+CDNは、2つの方法で待機時間を短縮します。ユーザーに近い場所からコンテンツを提供すること、2番目はエンドユーザーに近いTCP接続を終了することです。
+
+従来、ユーザーからバイトへの論理パスが短くなるように、CDNはバイトのキャッシュまたはコピーに使用されていました。多くの人が要求するファイルは、origin（サーバー）から一度取得してユーザーに近いサーバーへ保存できるため、転送時間を節約できます。
+
+CDNは、TCP遅延にも役立ちます。 TCPの遅延により、ブラウザーとサーバー間の接続を確立するのにかかる時間、接続を保護するのにかかる時間、および最終的にコンテンツをダウンロードする速度が決まります。せいぜいネットワークパケットは光の速度の約3分の2で移動するため、その往復にかかる時間は通信の両端がどれだけ離れているか、その間に何があるかによって決まります。混雑したネットワーク、過負荷の機器、ネットワークのタイプすべてがさらなる遅延を追加します。 CDNを使用して接続のサーバー側を訪問者の近くに移動すると、この遅延のペナルティが減少し、接続時間、TLSネゴシエーション時間が短縮されコンテンツのダウンロード速度が向上します。
+
+CDNは、多くの場合、訪問者の近くで静的コンテンツを保存および提供する単なるキャッシュと考えられていますが、さらに多くの機能を備えています。 CDNは、遅延のペナルティを克服するだけでなく、パフォーマンスとセキュリティの向上に役立つ他の機能を提供するものが増えています。
+
+* CDNを使用して動的コンテンツ（ベースHTMLページ、API応答など）をプロキシすることにより、ブラウザーとCDN自身のネットワークとの間の遅延が短縮され、元の速度に戻ることができます。
+* 一部のCDNは、ページを最適化してダウンロードとレンダリングを高速化する変換を提供するか、画像を最適化して表示するデバイスに適したサイズ（画像の大きさとファイルサイズの両方）に変換します。
+* [セキュリティ](./security)の観点から、悪意のあるトラフィックとボットは要求がoriginへ到達する前にCDNによって除外される可能性があり、その幅広い顧客ベースによりCDNが新しい脅威をより早く発見して対応できることを意味します。
+* [エッジコンピューティング](https://en.wikipedia.org/wiki/Edge_computing)の台頭により、サイトは訪問者の近くで独自のコードを実行できるようになりパフォーマンスが向上し、originの負荷が軽減されました。
+
+最後にCDNもまた、originサーバーがサポートしていない場合でもエッジからブラウザーまでHTTP/2、TLS1.3、またはIPv6を有効にできるなどoriginでの変更を必要とせずにサイトが新しいテクノロジーを採用できるようにします。
+
+### 警告と免責事項
+
+他の観察研究と同様に、測定できる範囲と影響には限界があります。 Web年鑑のCDN使用に関して収集された統計は、特定のCDNベンダーのパフォーマンスや有効性を意味するものではありません。
+
+Web年鑑に使用されるテスト[方法](./methodology)には多くの制限があります。これらには以下が含まれます。
+
+* **シミュレートされたネットワーク遅延**：Web年鑑は、[トラフィックを総合的に形成する](./methodology#webpagetest)専用ネットワーク接続を使用します。
+* **単一の地理的場所**：テストは[単一のデータセンター](https://httparchive.org/faq#how-is-the-data-gathered)から実行されるため、多くのCDNベンダーの地理的分布をテストすることはできません。
+* **キャッシュの有効性**：各CDNは独自の技術を使用しており、多くの場合、セキュリティ上の理由により、キャッシュのパフォーマンスは公開されていません。
+* **ローカリゼーションと国際化**：地理的分布と同様に言語、地理固有のドメインの影響もテストに対して不透明です。
```

Comment:
https://www.jtca.org/standardization/katakana_guide_3_20171222.pdf
```suggestion
* **ローカライゼーションと国際化**：地理的分布と同様に言語、地理固有のドメインの影響もテストに対して不透明です。
```

Reply:
I don't know this guideline, Thanks fixex.